### PR TITLE
[fix bug 1435065] Update /whatsnew Rocket page for ID locale.

### PR DESCRIPTION
**Pull request by jpetto**
## Description

Visual updates to the `/id/firefox/58.0/whatsnew/` page to push content up and add the "✅ Your Firefox is up to date" white banner up top.

## Issue / Bugzilla link

https://bugzilla.mozilla.org/show_bug.cgi?id=1435065

## Testing

Ensure no regressions.

### CODE REVIEW

File: media/css/firefox/whatsnew/whatsnew-rocket.scss [hunk 1]
```diff
@@ -8,45 +8,124 @@
 
 body {
     @include open-sans;
+}
+
+//* -------------------------------------------------------------------------- */
+// Up-to-date header
+
+.main-header {
```
Comment (jpetto) [inline media/css/firefox/whatsnew/whatsnew-rocket.scss]: For review purposes, this CSS is copied verbatim from `whatsnew.scss`.

File: bedrock/firefox/templates/firefox/whatsnew/index.id.html [hunk 2]
```diff
@@ -25,29 +25,39 @@
 
 {% block content %}
 <main role="main" class="section">
-  <div class="content">
-    <a class="moz-logo" href="{{ url('mozorg.home') }}"><img src="{{ static('img/logos/mozilla/wordmark-dark.svg')}}" alt="Mozilla" width="101" height="34"></a>
+  <header class="main-header">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/index.id.html]: Could this common header be an include that is shared between `/whatsnew` templates?

**Comment by alexgibson** [comment]
I'm guessing people are worried about the fold here, but visually this looks pretty cramped:

screen shot 2018-02-13 at 10 59 40

Could we perhaps use a vertical media query and give taller displays a little more breathing room? (don't want to rock the boat on this concept, I understand if we risk confusing approvals).

File: bedrock/firefox/templates/firefox/whatsnew/index.id.html (hunk 2 above)
Comment (jpetto) [reply bedrock/firefox/templates/firefox/whatsnew/index.id.html]: Good suggestion on making an `@import` and an `include`. Will do.

File: media/css/firefox/whatsnew/header.scss [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    text-align: center;
+
+    .inner-container {
+        position: relative;
+    }
+
+    h1 {
+        @include font-size-level5;
+        display: none;
+        margin: 7px 0;
```
Comment (alexgibson) [inline media/css/firefox/whatsnew/header.scss]: Suggest adding `font-weight; normal;` here? (It's bold on `/id/whatsnew/`, but not on the regular `/whatsnew/` page template).

Comment (jpetto) [reply media/css/firefox/whatsnew/header.scss]: Nice catch!

**Comment by jpetto** [comment]
@alexgibson Updated and squashed!

**Pull request merged**